### PR TITLE
Update detection rules landing page

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Updates the detection rules landing page

### Motivation
App sec beta + removing / updating outdated content

### Preview
https://docs-staging.datadoghq.com/sarina/detection-rules-update/security_platform/detection_rules/

### Notes
Putting this as DNM until we merge app sec docs

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/detection_rules/_index.md [hunk 1]
```diff
@@ -27,53 +25,45 @@ For each of monitoring option, there are [default detection rules][1] that work
 
 - With [Cloud Workload Security][5], the Datadog Agent actively monitors system activity and evaluates it against a set of detection rules.
 
-## Creating and managing detection rules
+- [Application Security][7] is enabled with the Datadog Agent and libraries to monitor application-level attacks aiming to exploit code-level vulnerabilities.
```
Comment (sarina-dd) [inline content/en/security_platform/detection_rules/_index.md:28]:
```suggestion
- [Application Security][7] leverages Datadog [APM][8], the [Datadog Agent][9], and in-app [detection rules][10] to detect threats in your application environment.
```

File: content/en/security_platform/detection_rules/_index.md [hunk 2]
```diff
@@ -85,3 +75,4 @@ Delete a rule by hovering over the rule and clicking the **Delete** button.
 [4]: /security_platform/cspm/
 [5]: /security_platform/cloud_workload_security/
 [6]: https://app.datadoghq.com/security/configuration/rules
+[7]: /security_platform/explorer/
```
Comment (sarina-dd) [inline content/en/security_platform/detection_rules/_index.md:78]:
```suggestion
[7]: /security_platform/explorer/
[8]: /tracing/
[9]: /agent/
```

File: content/en/security_platform/detection_rules/_index.md [hunk 3]
```diff
@@ -27,53 +25,45 @@ For each of monitoring option, there are [default detection rules][1] that work
...
 
 The free text search filters Detection Rules by text in the rule name or query. Query results update in real-time when the query is edited—there is no “Search” button to click.
 
 #### Filter by facet
 
-{{< img src="security_platform/security_monitoring/detection_rules/facets_panel.png" alt="Facets panel" width="75%">}}
+Use facets in the left panel to scope a search query by value. For example, if you have several rule types, such as `log detection` or `cloud configuration`, filter by `only` to see rules by rule type.
 
-Use facets in the left panel to scope a search query by value. For example, if you have several rule sources and need to troubleshoot on detection rules provided by one source, hover over a source value in the panel, such as `cloudtrail` or `kubernetes`, and click **only** to narrow the search to that source.
+{{< img src="security_platform/security_monitoring/detection_rules/rule_type_filter.png" alt="Filtering by rule type, such as a log detection or cloud configuration, in Datadog" >}}
```
Comment (jtappa) [inline content/en/security_platform/detection_rules/_index.md:42]: ❤️

File: content/en/security_platform/detection_rules/_index.md [hunk 4]
```diff
@@ -27,53 +25,45 @@ For each of monitoring option, there are [default detection rules][1] that work
...
-
-Clone a rule by hovering over the rule and clicking the **Clone** button.
+#### Rule and generated signal options
 
-#### Deleting a rule
+Click on the three dot menu, next to the rule toggle, and select any of the provided options: Edit, Clone, Delete, or View generated signals.
 
-Delete a rule by hovering over the rule and clicking the **Delete** button.
+- Click edit to update queries, adjust triggers, manage notifications or adjust rule configuration.
+  -  **Note**: You can only edit an out-of-the-box (OOTB) rule by first cloning the rule, and then modifying the rule. To edit a default rule, click on **Edit** and scroll to the bottom of the rule configuration page. Click **Clone**, and then modify the rule.
```
Comment (jtappa) [inline content/en/security_platform/detection_rules/_index.md:63]:
```suggestion
  -  **Note**: You can only edit an out-of-the-box (OOTB) rule by first cloning the rule, and then modifying the rule. To edit a default rule, click **Edit** and scroll to the bottom of the rule configuration page. Click **Clone**, and then modify the rule.
```

File: content/en/security_platform/detection_rules/_index.md [hunk 5]
```diff
@@ -27,53 +25,45 @@ For each of monitoring option, there are [default detection rules][1] that work
...
+- Clone a rule is helpful if you wish to duplicate an existing rule and lightly modify settings to cover other areas of detection. For example, duplicate a log detection rule and modify from **Threshold** to **Anomaly** to add new dimension to threat detection using the same queries and triggers.
```
Comment (jtappa) [inline content/en/security_platform/detection_rules/_index.md:64]:
```suggestion
- Cloning a rule is helpful if you wish to duplicate an existing rule and lightly modify settings to cover other areas of detection. For example, you could duplicate a log detection rule and modify it from **Threshold** to **Anomaly** to add new dimension to threat detection using the same queries and triggers.
```

**Review by jtappa — CHANGES_REQUESTED** [review]
small nits!

love your alt text! 😄

**Review by jtappa — APPROVED** [review]

**Pull request merged**